### ISSUE TITLE
[Web] [Accessibility] [11yMAS][HighImpact][Screen Readers][MAS2.4.4] [Link purpose is not defined for the Anchor links in the page.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#681]
**User Experience:**
User who rely on AT will not get proper information about “Anchor” links if  screen reader is not providing purpose of the link.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Developer tool

**Attachments**
 ![mas2 4 4_link purpose](https://user-images.githubusercontent.com/33054846/32368693-a6063356-c0ac-11e7-93e3-445f26b8ec50.png)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2. Navigate to “Anchor” link which is present before ”Additional resources” heading.
3. Verify Link purpose is defined for the “Anchor” link. 
 
**Current Behavior:** 
Link text is not provided for Anchor link, so screen reader is just announcing as Link.

**Expected Behavior:**
Alternative  text should be provided for anchor link which will behave like  link purpose as "Anchor" is an 
Image.

**Note:** 
1. Same Issue is reproducible for Anchor Links in the page.
2. Same issue is repro for the [MAC + safari][ Build info: OS Build: Mac OS Sierra 0.12.6 Browser: Safari 11.0.1] Link purpose is not defined for the anchor links in the page.
 
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={06a891af-210c-4c6d-bb3e-92babab209e7}&action=default 

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.4
